mlflow_cratedb/patch/mlflow/jobs/_job_runner.py

```
@@ -0,0 +1,51 @@
+
+if __name__ == "__main__":  # pragma: no cover
+    logger = logging.getLogger("mlflow.server.jobs._job_runner")
+    server_up_time = int(time.time() * 1000)
+    _start_watcher_to_kill_job_runner_if_mlflow_server_dies()
+
+    huey_store_path = os.environ[HUEY_STORAGE_PATH_ENV_VAR]
+
+    for job_name in _job_name_to_fn_fullname_map:
+        try:
+            _launch_huey_consumer(job_name)
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)